runtime/src/pallets_consensus.rs

```diff
@@ -210,63 +208,75 @@ impl pallet_session::Config for Runtime {
 }
 
+// TODO::Have to fine tune parameters for practical use-case
 #[cfg(feature = "with-staking")]
 parameter_types! {
-    // pub const SessionsPerEra: sp_staking::SessionIndex = 6;
-    // pub const BondingDuration: pallet_curveless_staking::EraIndex = 24 * 28;
-    pub const SlashDeferDuration: pallet_curveless_staking::EraIndex = 24 * 7; // 1/4 the bonding duration.
-    pub const MaxNominatorRewardedPerValidator: u32 = 256;
-    pub const ElectionLookahead: BlockNumber = constants::EPOCH_DURATION_IN_BLOCKS / 4;
-    pub const MaxIterations: u32 = 10;
-    // 0.05%. The higher the value, the more strict solution acceptance becomes.
-    pub MinSolutionScoreBump: Perbill = Perbill::from_rational_approximation(5u32, 10_000);
-    pub OffchainSolutionWeightLimit: Weight = constants::RuntimeBlockWeights::get()
-        .get(DispatchClass::Normal)
-        .max_extrinsic.expect("Normal extrinsics have a weight limit configured; qed")
-        .saturating_sub(BlockExecutionWeight::get());
+    // 27 Days
+    pub const SlashDeferDuration: sp_staking::SessionIndex = 27 * 6;
+    pub const MinSelectedValidators: u32 = 5;
+    pub const MaxNominatorsPerValidator: u32 = 25;
+    pub const MaxValidatorPerNominator: u32 = 25;
+    pub const DefaultValidatorFee: Perbill = Perbill::from_percent(20);
+    pub const DefaultSlashRewardProportion: Perbill = Perbill::from_percent(10);
+    pub const DefaultSlashRewardFraction: Perbill = Perbill::from_percent(50);
+    pub const MinValidatorStake: Balance = 10 * constants::MILLICENTS;
+    pub const MinNominatorStake: Balance = 5 * constants::MILLICENTS;
+    pub const MinNomination: Balance = 3 * constants::MILLICENTS;
+    pub const MaxChunkUnlock: usize = 32;
     pub const StakingPalletId: ModuleId = ModuleId(*b"mockstak");
-    /// We prioritize im-online heartbeats over election solution submission.
-    pub const StakingUnsignedPriority: TransactionPriority = TransactionPriority::max_value() / 2;
+    pub const StakingLockId: LockIdentifier = *b"staking ";
 }
-
 #[cfg(feature = "with-staking")]
-impl pallet_curveless_staking::Config for Runtime {
+impl pallet_nodle_staking::Config for Runtime {
+    type Event = Event;
     type Currency = Balances;
-    type UnixTime = Timestamp;
-    type CurrencyToVote = U128CurrencyToVote;
+    type BondedDuration = BondingDuration;
+    type MinSelectedValidators = MinSelectedValidators;
+    type MaxNominatorsPerValidator = MaxNominatorsPerValidator;
+    type MaxValidatorPerNominator = MaxValidatorPerNominator;
+    type DefaultValidatorFee = DefaultValidatorFee;
+    type DefaultSlashRewardProportion = DefaultSlashRewardProportion;
+    type DefaultSlashRewardFraction = DefaultSlashRewardFraction;
+    type MinValidatorStake = MinValidatorStake;
+    type MinValidatorPoolStake = MinValidatorStake;
+    type MinNominatorStake = MinNominatorStake;
+    type MinNomination = MinNomination;
     type RewardRemainder = ();
```

Comment:
```suggestion
    type RewardRemainder = CompanyReserve;
```

You may have to add some imports and stuff when applying the suggestion. But basically, we do not want to burn it and prefer to send remainders to the company reserve.